website/core/Footer.js

```diff
@@ -29,9 +29,14 @@ class Footer extends React.Component {
           </a>
         </section>
         <section className="finos finosBanner">
-          <a href="https://www.finos.org">
+          <div>
             <img id="finosicon" src={`https://alloy.finos.org/img/finos_wordmark.svg`} height='75px' alt="FINOS" title="FINOS"/>
-          </a>
+            <div>
+              <a class="nav-home" href="/">Alloy Documentation</a><br/>
+              <a class="nav-home" href="https://www.finos.org" target="_blank">FINOS website</a><br/>
```

Comment:
Do we need these first two links? The FINOS logo is already a link to the FINOS website. What if we just have Privacy Policy?

Reply:
Agreed. I shared the same feedback with @maoo last week. I think it should just be

- FINOS icon itself is a link to the FINOS website
- link to the privacy policy
- remove link to "Alloy Documentation (which is a circular anyway -- it's a link the site the site you're already on)"

Reply:
@maoo @gs-bracej I'd suggest we make the changes above before this PR is approved.